x-pack/plugins/task_manager/server/task_store.ts

```diff
@@ -171,11 +172,17 @@ export class TaskStore {
       );
     }
 
-    const savedObject = await this.savedObjectsRepository.create<SerializedConcreteTaskInstance>(
-      'task',
-      taskInstanceToAttributes(taskInstance),
-      { id: taskInstance.id, refresh: false }
-    );
+    let savedObject;
```

Comment:
I'll investigate this a bit. I'm not sure how the wrappers work entirely but I'll check. I think it's either done globally for all instances of SOC (which may be what you're thinking?) or can be done at an instance level 🤔

Reply:
I took a look at how this could be done. The saved objects client exposes a `addClientWrapper` but it wraps every instance of SOC created in the Kibana instance (including non task manager related ones). This would proxy every call made to the SOC which sounds worrisome to me. With that finding, I think I'll keep this iteration to what it is.